cli/types/cypress.d.ts

```diff
@@ -5131,6 +5137,21 @@ declare namespace Cypress {
     tag?: string
   }
 
+  interface DevServerOptions {
+    specs: Cypress.Cypress['spec'][]
+    config: {
+      supportFile: string
+      projectRoot: string
+      webpackDevServerPublicPathRoute: string
+    },
+    devServerEvents: NodeJS.EventEmitter,
+  }
+
+  interface ResolvedDevServerConfig {
+    port: number;
```

Comment:
nit: the rest of this file does not have `;`

Reply:
hehe right !! good eye